From 4e06515083d7effdee91195dbf93082e544a6283 Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Mon, 16 Feb 2026 13:58:08 -0500
Subject: [PATCH] convert: linux: properly match /etc/crypttab

I broke b227d493ace2250bbc53a25ce7db42fcf220ba51 during review
and neglected to retest. We need to match an augeas style
path when using starts_with

Fixes: https://issues.redhat.com/browse/RHEL-93583

Signed-off-by: Cole Robinson <crobinso@redhat.com>
(cherry picked from commit 109bc1ab3bd2f6b9d73616c546aeceb0631159be)
---
 convert/convert_linux.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/convert/convert_linux.ml b/convert/convert_linux.ml
index 4462e6cf..484148ab 100644
--- a/convert/convert_linux.ml
+++ b/convert/convert_linux.ml
@@ -1212,7 +1212,7 @@ fi
            This depends on /dev/sdXX in the guest having the same /dev/sdXX
            name in the appliance.
          *)
-        if String.starts_with "/etc/crypttab" path &&
+        if String.starts_with "/files/etc/crypttab" path &&
            String.starts_with "/dev/sd" value then (
           try
             let uuid = g#vfs_uuid value in
